src/css/_manager.scss

```diff
@@ -10,7 +10,7 @@
   margin-top: 4vh;
   margin-left: 250px;
   padding: 1.5em;
```

Comment:
I like the width, we should probably devote some attention to making this site responsive today.